charts/weave-policy-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: A Weaveworks Helm chart for Kubernetes to configure the policy agent
 name: weave-policy-agent
 appVersion: "1.0.0"
```

Comment:
Would be better if we upgrade the agent version as well, to keep it up to date

Reply:
the agent release blocked on token permission, would upgrade the version here once the agent new version is released.
Not going to merge this PR until the agent version is upgraded